postgres-consistency/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+# PR
```

Comment:
I think the module should be called: `importer-db-consistency` instead of `postgres-consistency`